provision-tahoe.py

```diff
@@ -18,13 +19,17 @@
 
 SRC_DIR = Path('/edx/src/')
 ENVS_DIR = SRC_DIR / 'edxapp-envs'
+PIP_DIR = SRC_DIR / 'edxapp-pip'
 EDXAPP_DIR = Path('/edx/app/edxapp')
 
 
 def move_environment_files_to_host():
     """
     Move the json environment files to the host so they're editable.
     """
+    if not ENVS_DIR.exists():
+        makedirs(ENVS_DIR)
```

Comment:
Are we using Python 3 on this yet? If we are, we can do `makedirs(ENVS_DIR, exist_ok=True)`

Reply:
Nope, not really.